src/components/StorySection.js

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { containsHtml, trimNewLines } from '../helpers';
+import { Button } from './Button';
+import { HtmlView } from './HtmlView';
+import { ImageSection } from './ImageSection';
+import { MediaSection } from './MediaSection';
+import { BoldText } from './Text';
+import { Wrapper, WrapperHorizontal } from './Wrapper';
+
+// eslint-disable-next-line complexity
+export const StorySection = ({ contentBlock, index, openWebScreen, settings }) => {
+  return (
+    <>
+      {index > 0 && !!contentBlock.title && (
```

Comment:
please take the comments with the refactoring.

>// skip the title for the first content block because it is used as the main title

otherwise it will be harder to remember, why we skip the first index

Reply:
added it again